src/test/java/lottoAuto/model/WinningLottoTest.java

```diff
@@ -0,0 +1,57 @@
+package lottoAuto.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class WinningLottoTest {
+    private WinningLotto winningLottoTest;
+
+    @BeforeEach
+    public void set() {
+        Set<LottoNumber> winningNumbers = new HashSet<>();
+        winningNumbers.add(new LottoNumber(1));
+        winningNumbers.add(new LottoNumber(2));
+        winningNumbers.add(new LottoNumber(3));
+        winningNumbers.add(new LottoNumber(4));
+        winningNumbers.add(new LottoNumber(5));
+        winningNumbers.add(new LottoNumber(6));
```

Comment:
자바의 `Collection`은 `복사 생성자(또는 변환 생성자)`라는 녀석을 제공하고 있는데요~ 아래와 같이 변경해볼 수 있을 것 같아요. :)
```suggestion
        List<LottoNumber> lottoNumberList = Arrays.asList(
                new LottoNumber(1),
                new LottoNumber(2),
                new LottoNumber(3),
                new LottoNumber(4),
                new LottoNumber(5),
                new LottoNumber(7)
        );
        Set<LottoNumber> lottoNumbers = new HashSet<>(lottoNumberList);
```